bot/exts/holidays/halloween/scarymovie.py

```diff
@@ -44,10 +51,16 @@ async def select_movie(self) -> dict:
             total_pages = data.get("total_pages")
 
         # Get movie details from one random result on a random page
-        params["page"] = random.randint(1, total_pages)
+        params["page"] = min(random.randint(1, total_pages), 500)
```

Comment:
While this works the result is no longer random for some inputs. Wouldn't something like this be better?
```suggestion
        params["page"] = random.randint(1, min(total_pages, 500))
```